sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -628,7 +628,17 @@ public void init(CosmosClientMetadataCachesSnapshot metadataCachesSnapshot, Func
                     this.clientTelemetryConfig,
                     this,
                     this.connectionPolicy.getPreferredRegions());
-            clientTelemetry.init();
+            clientTelemetry.init().thenEmpty((publisher) -> {
+                logger.info(
```

Comment:
when client telemetry is enabled, this log may never got logged.

`sendClientTelemetry` will repeated calling itself, so the thenEmpty may never be called (I think)

Reply:
Fixed